### DISCUSSION TITLE
Library State

### DISCUSSION THREAD

**Discussion by DevNico** [espresso3389/flutter_pdf_render#119] [General]
Hey @espresso3389 first of all great library!

Even though the package is great it does seem to have some limitations which would require further development. Especially regarding #104, #107 and #93. 

Are you actively working or planning to work on these issues? Obviously life happens and sometimes you just lose interest in a project so this is just a general inquiry on the state of development / future plans.

**Comment by espresso3389** [comment]
Simply, I've been busy... :(
I understand everyone is waiting for such improvements but no time for the project just now.

**Reply by DevNico** [reply]
No worries like I said life happens :)

Just so everyone waiting can make an informed decision on wether or not to use your package, do you see a time in the near future when you are going to continue development?

**Comment by JakobStadlhuber** [comment]
Hey @espresso3389 could you probably activate GitHub Sponsor https://github.com/sponsors it is available in japan as far as I can see. We would be pleased to support the project especially if the project could make rendering calls async https://github.com/espresso3389/flutter_pdf_render/issues/107

Wish you the best
